test/templates/pod_anp.go

```diff
@@ -47,16 +47,16 @@ spec:
     - externalEntitySelector:
         matchLabels:
 {{ if .To.Entity.Kind }}
-          kind.nephe: {{.To.Entity.Kind}}
+          nephe.io/kind: {{.To.Entity.Kind}}
```

Comment:
@shenmo3 Can you evaluate if labels can you substituted from constant variables of package, rather than hardcoded it here in template.

Reply:
I believe its possible, will file an issue and take it later.

Reply:
Could you do it in this PR itself ?